Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md

```diff
@@ -10,6 +10,22 @@ On Umbraco Cloud, users work in almost the same way as on a normal installation
 
 In this article, we will show how users work, as well as explain the different settings for users on Umbraco Cloud.
 
+## Umbraco ID
+
+On Umbraco CLoud project we use Umbraco ID for new projects and projects on 8.9.1 and above.
```

Comment:
minor typo: "Umbraco CLoud"
Should be "Umbraco Cloud" with lower case L

Reply:
Good catch, it has been fixed